README.md

```diff
@@ -61,6 +61,9 @@ The fallback content in case of a fetch error or unsupported browser.
 **cacheRequests** {boolean} ▶︎ `true`  
 Cache remote SVGs.
 
+**fetchOptions** {string}  
```

Comment:
Can you move this new prop after the `description`? To keep them ordered alphabetically.
And change the type from `string` to `RequestInit` and the text to `Custom options for the [request](https://developer.mozilla.org/en-US/docs/Web/API/Request/Request).`